### PR TITLE
Add boolean to DataMappingSmokeTest

### PR DESCRIPTION
**Pull request by zaz968m**

### CODE REVIEW

**Comment by kokosing** [comment]
Ping me once automation completes

**Review by kokosing — APPROVED** [review]

**Review by Praveen2112 — APPROVED** [review]

**Comment by kokosing** [comment]
Merged, thanks!

File: presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java [hunk 1]
```diff
@@ -146,6 +146,11 @@ protected boolean isColumnNameRejected(Exception exception, String columnName, b
             return Optional.empty();
         }
 
+        if (typeName.equals("boolean")) {
+            // MySql does not have built-in support for boolean type. MySQL provides BOOLEAN as the synonym of TINYINT(1)
+            return Optional.empty();
```
Comment (findepi) [inline presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java:151]: This probably means we should map boolean into tinyint in mysqlclient.
Can you file an issue?

Comment (zaz968m) [reply presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java:151]: done.

**Pull request merged**